mesheryctl/internal/cli/root/system/error.go

```diff
@@ -63,7 +63,7 @@ func ErrCreateDir(err error, obj string) error {
 }
 
 func ErrUnmarshal(err error, obj string) error {
-	return errors.New(ErrUnmarshalCode, errors.Alert, []string{"Error unmarshalling file ", obj}, []string{err.Error()}, []string{}, []string{})
+	return errors.New(ErrUnmarshalCode, errors.Alert, []string{"Error processing JSON response from server to file", obj}, []string{err.Error()}, []string{}, []string{})
```

Comment:
`processing JSON response from server to file`

Seems a little weird if we aren't unmarshalling files. Is this only handling files? If not just omit `to file`

Reply:
done 👍